lib/gringotts/gateways/monei.ex

```diff
@@ -94,10 +148,21 @@ defmodule Gringotts.Gateways.Monei do
   * `capture/3` _an_ amount.
   * `void/2` a pre-authorization.
 
-  ### Note  
+  ## Note  
   
   A stand-alone pre-authorization [expires in
   72hrs](https://docs.monei.net/tutorials/manage-payments/backoffice).
+
+  ## Example
+
+  The following session shows how one would (pre) authorize a payment of $40 on the sample `card`.
+
+      iex> auth_result = Gringotts.authorize(:payment_worker, Gringotts.Gateways.Monei, 40, card, opts)
```

Comment:
in the example, card parameter is not defined and same for opts.
The example should mimic the same behaviour as someone is going to use it.

card = %CreditCard{}
opts = []
amount = 40

iex> auth_result = Gringotts.authorize(:payment_worker, Gringotts.Gateways.Monei, amount, card, opts)

Reply:
Please refer lines [163](https://github.com/aviabird/gringotts/pull/38/files#diff-cd577a67abd155af0d2d475f678770c1R163) (right below) and [85](https://github.com/aviabird/gringotts/pull/38/files#diff-cd577a67abd155af0d2d475f678770c1R85).

Reply:
ideally this should be present before each example, as it helps developers to understand the examples better.

Reply:
Done!